### PR TITLE
Use default settings for Brave Ads UserActivityStudy for 10% of users - Production

### PR DESCRIPTION
**Pull request by tmancey**
Uplift of https://github.com/brave/brave-variations/pull/208

### CODE REVIEW

**Comment by tmancey** [comment]
See https://github.com/brave/brave-core/pull/7881 for STR/Cases

**Comment by btlechowski** [comment]
LGTM

### BraveAds.UserActivityStudy:Triggers=EMPTY/Threshold=0.0/IdleTimeThreshold=5

image

Verified Threshold is 0
```
Triggered event: 01 (2.5:0:3600 s)
```

Verified IdleTimeThreshold is 5
```
[28006:28006:0127/071223.452914:VERBOSE1:ads_impl.cc(230)] Browser state changed to unidle after 5 s
```

Verified ad was shown.
image

### BraveAds.UserActivityStudy:IdleTimeThreshold=5

image


Verified Threshold is default 2
```
[5507:5507:0127/084047.295956:VERBOSE6:user_activity.cc(44)] Triggered event: 02 (4:2:3600 s)
```

Verified IdleTimeThreshold is 5
```
[5507:5507:0127/084439.627998:VERBOSE1:ads_impl.cc(214)] Browser state changed to idle
[5507:5507:0127/084440.644494:VERBOSE1:ads_impl.cc(230)] Browser state changed to unidle after 5 s
```

Verified ad was shown:
image

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on `main` (staging) as per https://github.com/brave/brave-variations/pull/211#issuecomment-1023341486.

**Pull request merged**
